spec/lucky/static_compression_handler_spec.cr

```diff
@@ -0,0 +1,107 @@
+require "../spec_helper"
+require "http"
+
+include ContextHelper
+
+private PATH = "example.css"
+
+describe Lucky::StaticCompressionHandler do
+  it "calls next when not enabled" do
+    context = build_context(path: PATH)
+    context.request.headers["Accept-Encoding"] = "gzip"
+    next_called = false
+
+    call_handler_with(context) { next_called = true }
+
+    next_called.should be_true
```

Comment:
I believe this spec and the next one should also assert that the content was not gzipped by doing something like: `context.response.headers["Content-Encoding"].should_not eq "gzip"`